librosa/display.py

```diff
@@ -509,6 +509,56 @@ def __init__(self, times, y, steps, envelope, sr=22050, max_samples=11025):
         self.envelope = envelope
         self.sr = sr
         self.max_samples = max_samples
+        self.cid = None
+        self.ax = None
+
+    def __del__(self):
+        self.disconnect(strict=True)
+
+    def connect(self, ax, *, signal="xlim_changed"):
+        """Connect the adaptor to a signal on an axes object.
+
+        Note that if the adaptor has already been connected to an axes object,
+        that connect is first broken and then replaced by a new callback.
+
+        Parameters
+        ----------
+        ax : matplotlib.axes.Axes
+            The axes to connect with this adaptor's `update`
+        signal : string, {"xlim_changed", "ylim_changed"}
+            The signal to connect
+
+        See Also
+        --------
+        disconnect
+        matplotlib.axes.callbacks.connect
```

Comment:
i am not finding the documentation for this

closest i found is
https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.add_callback.html

Reply:
Ah I see; it should be the connect method of `CallbackRegistry`.

Reply:
I'll actually just remove this see-also, since it's not necessary or helpful in the abstract.